dias/config_loader.py

```diff
@@ -104,10 +130,7 @@ def _check_config_variable(self, key, proptype, default=None, config=None):
                                       .format(value, key, proptype))
 
     def _load_analyzers(self, limit_task):
-        """
-        Locate and load all task config files
-        """
-
+        """Locate and load all task config files."""
```

Comment:
Replace with:
```
Instantiate all :class:Task objects by locating and loading all task config files.
The tasks are stored in a list as `self.tasks`.

Parameters
--------------
   limit_task : string or None
       If not None, only the task named `limit_task` will be loaded: all other tasks are ignored.
```

Reply:
Same here: I added your test for the header to stay a single line.

Also: this private function doesn't make it to the html API reference either. But we could change that...